dartagnan/src/main/java/com/dat3m/dartagnan/program/processing/compilation/VisitorC11.java

```diff
@@ -63,7 +63,7 @@ public List<Event> visitStart(Start e) {
         return tagList(eventSequence(
                 load,
                 super.visitStart(e),
-                newJump(expressions.makeNotEqual(resultRegister, one), (Label) e.getThread().getExit())));
+                newJumpUnless(expressions.makeNotEqual(resultRegister, one), (Label) e.getThread().getExit())));
```

Comment:
Isn't this wrong? The semantics is "if I don't read 1 (create was done), then jump"